src/Illuminate/Support/Collection.php

```diff
@@ -726,6 +726,24 @@ public function forPage($page, $perPage)
     }
 
     /**
+     * Returns an array with two elements. Items in the first element did pass
```

Comment:
I'm afraid I don't see the formatting problem. Could you tell what's wrong with the current formatting?